src/data/sym/basic.lean

```diff
@@ -232,10 +232,14 @@ by simp [sym.map, subtype.mk.inj_eq]
   (a :: s).map f = (f a) :: s.map f :=
 by { cases s, simp [map, cons] }
 
+@[congr] lemma map_congr {β : Type*} {f g : α → β} {s : sym α n} :
+  (∀ x ∈ s, f x = g x) → map f s = map g s :=
+λ h, by simpa [map, subtype.mk.inj_eq] using multiset.map_congr h
```

Comment:
If the proof starts with `λ h` you might as well move that assumption left of the `:` in the statement.